gnowledge/gstudio
# Course player Part - 1

Course Player is the dashboard of CourseEvent Group.

Implemenation:
A Course has Section, Subsection, Unit, Resources
A CourseEvent Group has SectionEvent, SubsectionEvent, Forum, Thread.
Course Player uses forums and threads.
It also has discussion course-wide
It also has Journal, adding blog pages.


What to test:
1.Create a Course.
2. Author it. Add resources with images,videos, pages with contents.
3. Announce it.
4. Go to CourseEVentGroup.
5. Forums i.e Units are listed in cards.
6. Click on any one of them. You will be able to see other forums of the group on LHS.
7.All the threads of respective forum will be displayed. If resource in course unit was image/video, it is added to content of thread.
8. Click on any thread, you will be able to see other threads of the forum on LHS.
9. Reply on thread.
10. CourseEvent group discussion tab. Try to add posts on it.
11. Journal tab. Add blog pages

Meanwhile, test forums and threads in regular groups. 
Minor bugs fixed in Forum and Threads.
